guides/common/modules/ref_accessing-ansible-modules-docs.adoc

```diff
@@ -0,0 +1,14 @@
+[id="accessing-ansible-modules-docs_{context}"]
+== Accessing {Project} Ansible Modules Documentation
+
+You can read the content collection documentation offline on the command line using the following command:
+
+----
+# ansible-doc redhat.satellite.[module_name]
```

Comment:
What's the point of this procedure? The same step is presented in the listing procedure already.

Reply:
Not a proc but just a ref section for offline and online docs.